src/plugins/shared_ux/public/components/redirect_app_links/redirect_app_links.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FunctionComponent, useRef, useMemo } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { ApplicationStart } from 'src/core/public';
+import { createNavigateToUrlClickHandler } from './click_handler';
+
+export interface RedirectCrossAppLinksProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
I'd try to stay consistent with the naming of the component vs it's props

```suggestion
export interface RedirectAppLinksProps extends React.HTMLAttributes<HTMLDivElement> {
```

Reply:
Ah good catch thanks! [f11aa17](https://github.com/elastic/kibana/pull/124197/commits/f11aa17f3f0f14a06e6ed56d62c481326d20a2c0)